src/screens/UserPortal/UserGlobalScreen/UserGlobalScreen.module.css

```
@@ -15,33 +13,41 @@
 
 @keyframes moveLeft {
```

Stylelint (17.6.0)

[error] Expected keyframe name "moveLeft" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)